unity/README.md

```diff
@@ -25,3 +25,14 @@ To get started on building the document:
 2. Change directory into `Documentation` and run either `scripts/build.cmd` or `scripts/build.sh`
 
 (If you want to have the web page served locally, instead of the script run `docfx docfx.json --serve`)
+
+# FAQ
+### (*1) I encountered a problem with building the android application/running it on my device. How do I resolve it?
```

Comment:
I was thinking about a more specific Q&A. How about you break them down into two questions?

1. I cannot build the project for Android.
    1. In Build Settings, add Scenes/DeviceData to the scenes in Build.
1. I only see a black screen when opening the app.
    1. Other player settings...

Reply:
got it!

Reply:
Oh I think it should be the other way around, i.e. black screen --> add scene, cannot build --> change build settings

Reply:
Oh, you are right!